tools/wheels/test.f

```diff
@@ -1,3 +0,0 @@
-      program first
-      print *,'This is my first program'
-      end program first
```

Comment:
`tools/wheels/cibw_before_build_macos.sh` is still calling `gfortran tools/wheels/test.f` in this branch--are you planning to delete that later or what is going on with that?

Reply:
Good catch, I missed that. It looks like this was accidentally committed in commit 200a8169fd0 after testing something, because it doesn't do anything nor is there a relevant comment anywhere. So I think I'll push a cleanup, unless @andyfaff you actually intended for this test file to stay?

Reply:
I added that file when I wanted to be sure that the Fortran compiler was working. I think it's used in cibw_before_build_macos.sh.

Without the file it was hard to diagnose compiler and runtime issues later in the wheel build stage. Its not necessary to have now.

Reply:
If you remove it, you'll need to modify the cibw script otherwise wheel builds won't work

Reply:
Thanks for the context. It doesn't seem needed anymore and we don't have dummy compiles like this in other build scripts, so I added a commit to remove it.